plugins/tutor-contrib-paragon/tutorparagon/patches/mfe-k8s-volumes

```diff
@@ -0,0 +1,13 @@
+{% if MFE_HOST_EXTRA_FILES %}
```

Comment:
This check seems redundant, as it's [already in `k8s-deployments`](https://github.com/overhangio/tutor-mfe/blob/main/tutormfe/patches/k8s-deployments#L33-L33).